js/src/sdk/models/connectedAccounts.ts

```diff
@@ -1,27 +1,36 @@
 
-import {  InitiateConnectionPayloadDto, GetConnectionsResponseDto, GetConnectionInfoData, GetConnectionInfoResponse, GetConnectionsData, InitiateConnectionData } from "../client";
+import { InitiateConnectionPayloadDto, GetConnectionsResponseDto, GetConnectionInfoData, GetConnectionInfoResponse, GetConnectionsData, InitiateConnectionData } from "../client";
 import client from "../client/client";
 import apiClient from "../client/client"
 import { BackendClient } from "./backendClient";
+import { Integrations } from "./integrations";
+import { Apps } from "./apps";
 import { CEG } from "../utils/error";
 
-type ConnectedAccountsListData = GetConnectionsData['query'] & {appNames?: string};
+type ConnectedAccountsListData = GetConnectionsData['query'] & { appNames?: string };
 
 type InitiateConnectionDataReq = InitiateConnectionPayloadDto & {
     data?: Record<string, unknown> | unknown;
     entityId?: string;
     labels?: string[];
-    integrationId: string;
+    integrationId?: string;
     redirectUri?: string;
+    authMode?: string;
+    authConfig?: { [key: string]: any },
```

Comment:
The `authConfig` type is using `any` which could lead to runtime errors. Consider defining a proper interface for better type safety:
```typescript
interface AuthConfig {
    clientId?: string;
    clientSecret?: string;
    [key: string]: string | undefined;
}
```

Reply:
It's not required here.